File: backend/check_models.py
import os
import requests
from dotenv import load_dotenv

load_dotenv()
api_key = os.environ.get("GROQ_API_KEY")

headers = {
    "Authorization": f"Bearer {api_key}",
    "Content-Type": "application/json"
}
response = requests.get("https://api.groq.com/openai/v1/models", headers=headers)
print(response.json())

File: backend/check_models.js
require("dotenv").config();
const Groq = require("groq-sdk");
const groq = new Groq({ apiKey: process.env.GROQ_API_KEY });
async function main() {
  const models = await groq.models.list();
  console.log(models.data.map(m => m.id));
}
main();
